ocw_data_parser/ocw_data_parser_test.py

```diff
@@ -302,6 +302,7 @@ def test_course_files(ocw_parser):
         'platform_requirements': '',
         'description': '',
         'type': 'OCWImage',
+        'file_location': '182.png'
```

Comment:
Can we add a test which uses the `bucket_base_url` branch in `_compose_media_dict`?